doc/classes/EditorSettings.xml

```diff
@@ -1593,6 +1597,10 @@
 		<member name="text_editor/theme/highlighting/gdscript/annotation_color" type="Color" setter="" getter="">
 			The GDScript syntax highlighter text color for annotations (e.g. [code]@export[/code]).
 		</member>
+		<member name="text_editor/theme/highlighting/gdscript/bracket_pair_colors" type="PackedColorArray" setter="" getter="">
+			The list of colors to cycle through for colorizing bracket pairs.
+			Leave blank to disable.
```

Comment:
Needs some description of how it works, how it knows what color to use for what bracket pair